v3.0-RC2/station_information.json

```
@@ -0,0 +1,276 @@
+                        }
+                      }
+                    }
+                  }
+                }
+              },
+              "parking_type": {
+                "description": "Type of parking station. Added in v2.3",
+                  "type": "string",
+                  "enum": [
+                    "parking_lot",
+                    "street_parking",
+                    "underground_parking",
+                    "sidewalk_parking",
+                    "other"
+                  ]
+                },
+              "parking_hoop": {
+                "description": "Are parking hoops present at this station? Added in v2.3",
+                "type": "boolean"
+              },
+              "contact_phone": {
+                "description": "Contact phone of the station. Added in v2.3",
+                "type": "string"
+              },
+              "capacity": {
+                "description":
+                  "Number of total docking points installed at this station, both available and unavailable.",
+                "type": "integer",
+                "minimum": 0
+              },
+              "vehicle_types_capacity": {
+                "description": "This field's value is an array of objects containing the keys vehicle_type_ids and count defined below. These objects are used to model the parking capacity of virtual stations (defined using the is_virtual_station field) for each vehicle type that can be returned to this station.",
+                "type": "array",
+                "items": {
+                  "type": "object",
+                  "properties": {
+                    "vehicle_type_ids": {
+                      "description": "A vehicle_type_id, as defined in vehicle_types.json, that may park at the virtual station.",
+                      "type": "string"
+                    },
+                    "count": {
+                      "description": "A number representing the total number of vehicles of the corresponding vehicle_type_id type that can park within the virtual station.",
+                      "type": "integer",
+                      "minimum": 0
+                    }
+                  },
+                  "required": ["vehicle_type_id","count"]
+                }
+              },
+              "vehicle_docks_capacity": {
+                "description": "This field's value is an array of objects containing the keys vehicle_type_id and count defined below. These objects are used to model the total docking capacity of a station, both available and unavailable, for each type of vehicle defined in vehicle_types.json.",
+                "type": "array",
+                "items": {
+                  "type": "object",
+                  "properties": {
+                    "vehicle_type_ids": {
+                      "description": "A vehicle_type_id, as defined in vehicle_types.json, that may dock at the station.",
+                      "type": "string"
```

Let's please change the type of `vehicle_type_ids` to an array of strings as defined in https://github.com/MobilityData/gbfs/pull/547.

Thank you!

```suggestion
                      "type": "array",
                      "items": {
                        "type": "string"
                      }
```